content/docs/capabilities/getting-users-identity.md

```diff
@@ -139,7 +140,7 @@ Though you will likely verify signed headers programmatically in your applicatio
 
 1. Finally, we want to cryptographically verify the validity of the token. To do this, we will need the signer's public key. You can simply copy and past the output of `cat ec_public.pem`.
 
-   ![httpbin displaying verified jwt](./img/verifying-headers-2.png)
+   ![httpbin displaying verified jwt](./img/jwt-payload.png)
```

Comment:
I wonder if we should update the other two screenshots as well (and maybe the instructions too — I'm not sure our deploy examples include httpbin anymore, so I don't know whether people reading this would be able to follow along easily).

If you think this is worthwhile maybe you could file a follow-up issue? I'd be happy to help.

Reply:
Thanks for filing an issue! Sounds good to me.